code/obj/item/book.dm

```diff
@@ -3361,7 +3361,7 @@ On that note, CentComm would like to remind trainees that they do not recieve cl
 		Damaged windows may be repaired by applying glass to a grille; if the grille is damaged or missing, repair it with metal rods from sheets or storage.
 		<br>
 		<br>
-		Once all breaches have been repaired, obtain and utilize an air canister (white, potentially with a red stripe) and space HVAC on heat setting.
+		Once all breaches have been repaired, obtain and utilize an air canister (white, potentially with a red stripe) and space HVAC on heat setting.  Alternatively utilize an oxygen grenade ensuring no associates are in blast zone prior to activation.
```

Comment:
```suggestion
		Once all breaches have been repaired, obtain and utilize an air canister (white, potentially with a red stripe) and space HVAC on heat setting.  Alternatively utilize an oxygen grenade ensuring no staff members are in the blast zone prior to activation.
```
@nefarious6th thoughts?

Reply:
Up to you! Missed this earlier, I think "associates" has the right detached and non-caringness that might come with an outside firm (Servin Underwriting) preparing the instructional documents. That gets into the weeds on it though, I think either works fine.